src/pages/Facility/settings/billing/discount/discount-codes/CreateDiscountCodeSheet.tsx

```diff
@@ -25,8 +25,8 @@ export function CreateDiscountCodeSheet() {
   const [open, setOpen] = useState(false);
 
   const { mutate: createCode } = useMutation({
-    mutationFn: mutate(facilityApi.updateMonetaryComponents, {
-      pathParams: { facilityId },
+    mutationFn: mutate(facilityApi.setMonetaryComponents, {
+      pathParams: { id: facilityId },
```

Comment:
This would've been better left as facilityId, and the pathparam also left as facilityId